packages/clayui.com/src/pages/docs/advanced_guides/using-js-components-with-metal.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Using JS Components with Metal.js"
+description: ""
+layout: "guide"
+weight: 1
+---
+
+<article id="using-js-components-with-metal">
+
+Clay Components is made according to Lexicon standards and built on top of the [Metal.js](https://metaljs.com/) framework, so you can take advantage of Clay's components in your two template systems.
+
+## Jsx
+
+You can import your components as described in [Importing the JS Component](/docs/getting_started/importing-the-js-component.html), consider:
+
+
+```javascript
+import {ClayBadge} from 'clay';
+```
+
+In JSX components call only one component of Clay as any JSX component in Metal.js, consider:
+
+```javascript
+import Component from 'metal-jsx';
+import {ClayBadge} from 'clay';
+
+class App extends Component {
+    render() {
+        return(
+            <div>
+                <h4>My Application with Clay<h4>
+                <hr />
+                <ClayBadge
+                    id="myId"
+                    label="badge"
+                    style="danger"
+                />
+            </div>
+        );
+    }
+};
+```
+
+## Soy
+
+In SOY components you can call the components of Clay as any SOY component. Make sure you import the Clay component where you want to use it. Consider:
```

Comment:
I think the `make sure you import...` line needs to be more relevant... maybe in bold or in a `warn` div that somehow makes sure you won't miss it ;)

Reply:
Ah yeah, better put a `warn` to ensure 🙂. Thanks!